pkgs/applications/misc/ethereum/cpp/default.nix

```diff
@@ -0,0 +1,134 @@
+{ stdenv
+, fetchgit
+, cmake
+, boost
+, gmp
+, jsoncpp
+, leveldb
+, cryptopp
+, cudatoolkit
+, nvidia_x11
```

Comment:
Isn't there a way to mine with just CPU? This could be another flag if it truly is optional.